configs/simple_copy_paste/mask-rcnn_r50_fpn_rpn-2conv_4conv1fc_syncbn-all_32xb2-ssj-270k_coco.py

```diff
@@ -11,7 +11,7 @@
 norm_cfg = dict(type='SyncBN', requires_grad=True)
 # Use MMSyncBN that handles empty tensor in head. It can be changed to
 # SyncBN after https://github.com/pytorch/pytorch/issues/36530 is fixed
-# Requires MMCV-full after  https://github.com/open-mmlab/mmcv/pull/1205.
+# Requires MMCV after  https://github.com/open-mmlab/mmcv/pull/1205.
```

Comment:
we can delete this line of comments because mmdet 3.x relies on mmcv 2.x, which must have support it.